src/Entity/Address.php

```diff
@@ -49,9 +49,20 @@ class Address
      */
     private $city;
 
-    public function __construct()
+    public function __construct(
+        string $name,
```

Comment:
Try to make types more specific using Value Objects if possible

Reply:
Is there any specific library that you would prefer I use?

Reply:
I would love to have just plain PHP objects for that, and probably use https://github.com/beberlei/assert for validation